src/main/java/it/pagopa/transactions/client/NodeForPspClient.java

```diff
@@ -81,10 +88,11 @@ public Mono<ClosePaymentResponseDto> closePaymentV2(ClosePaymentRequestV2Dto req
         );
         return nodoWebClient.post()
                 .uri(
-                        uriBuilder -> uriBuilder.path("/nodo/nodo-per-pm/v2/closepayment")
+                        uriBuilder -> uriBuilder.path("/nodo-auth/nodo-per-pm/v2/closepayment")
```

Comment:
I would parameterized (maybe in another PR) this path in order to change it's value upgrading helm instead of modifying code. What do you think?